gift_card.xml

```diff
@@ -87,6 +86,23 @@
             <field name="action" ref="gift_card_redeem_wizard"/>
             <field name="group" ref="account.group_account_admin"/>
         </record>
-
+        <record model="ir.action.act_window.domain" id="act_gift_card_form_domain_draft">
+            <field name="name">Draft</field>
+            <field name="sequence" eval="10"/>
+            <field name="domain">[('state', '=', 'draft')]</field>
+            <field name="act_window" ref="act_gift_card"/>
+        </record>
+        <record model="ir.action.act_window.domain" id="act_gift_card_form_domain_active">
+            <field name="name">Active</field>
+            <field name="sequence" eval="10"/>
```

Comment:
The `sequence` `eval` will be different for all 3 tabs. Keep them in increasing order.